.devcontainer/scripts/container-creation.sh

```diff
@@ -9,3 +9,6 @@ git submodule update --init --recursive
 # Also run the full restore on the repo so that go-to definition
 # and other language features will be available in C# files
 ./restore.sh
+# The container creation script is executed in a new Bash instance
+# so we exit at the end to avoid the creation process lingering.
+exit
```

Comment:
Maybe my bash is rusty, but don't scripts implicitly exit their host process when they reach the end?

Reply:
Yep -- it does. This is more so working around some funkiness with the way Codespaces seems to launch the `postCommand` script.